class2.html

```diff
@@ -360,15 +360,15 @@ <h2>What is CSS?</h2>
           </p>
           <p>
             <span class="bold">Style Sheets</span>
-            - refers to "style sheet language".<br>
+            - refers to a file <em>containing</em> CSS code.<br>
             This language is used to describe the presentation of a web page.
           </p>
         </section>
 
         <section>
           <h2>What can CSS do?</h2>
           <p>CSS can change colors, fonts, and the position of content.</p>
-          <a href="https://www.w3schools.com/css/css_intro.asp" target="_blank">
+          <a href="http://www.mezzoblue.com/zengarden/alldesigns/" target="_blank">
```

Comment:
I like the zengarden examples 👍 Thoughts on removing the `target="_blank"`?

Reply:
Good call, seeing as we discourage it.